db/seeds.rb

```diff
@@ -11,3 +11,8 @@
               { name: 'skill4', featured: true },
               { name: 'skill5', featured: false },
              ])
+
+Group.create([{ name: '1409' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.